.devcontainer/docker-compose.yaml

```diff
@@ -27,6 +27,9 @@ services:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: postgres
 
+    ports:
+      - "5432:5432"
+    # For local developemnt, we need to forward the database port here too.
     # Add "forwardPorts": ["5432"] to **devcontainer.json** to forward PostgreSQL locally.
     # (Adding the "ports" property to this file will not forward from a Codespace.)
```

Comment:
This comment maybe true for Codespaces but it's not true for local devContainer.
The database is not accessible unless you forward it's port here too.